xcode/Subconscious/Shared/Components/Deck/DeckView.swift

```diff
@@ -143,6 +148,11 @@ extension AppAction {
             return .assignColor(address: address, color: color)
         case let .requestAppendToEntry(address, append):
             return .appendToEntry(address: address, append: append)
+        case let .requestUpdateLikeStatus(address, liked):
```

Comment:
Nit: could we coalesce these three actions into one action that takes a boolean?

`setLike(_ isLiked: Bool)`